src/components/sporsmal/typer/periode-komp.tsx

```diff
@@ -20,15 +21,27 @@ type AllProps = SpmProps & PeriodeProps;
 
 const PeriodeKomp = ({ sporsmal, index, slettPeriode }: AllProps) => {
     const { setValue, errors, getValues } = useFormContext()
+    const [ datoer, settDatoer ] = useState<Date[]>([])
     const id = sporsmal.id + '_' + index
     const htmlfor = sporsmal.id + '_t_' + index
     const feilmelding = hentFeilmelding(sporsmal)
+    Norwegian.rangeSeparator = ' - '
 
     useEffect(() => {
         setValue(id, hentPeriode(sporsmal, index))
         // eslint-disable-next-line
     }, [ sporsmal ]);
 
+    const onValueUpdate = (selectedDates: Date[]) => {
+        if (selectedDates.length > 0) {
+            settDatoer(selectedDates)
+        }
+    }
+
+    const formater = () => {
+        return datoer.map(dato => dayjs(dato).format('DD.MM.YYYY')).join('   -    ')
```

Comment:
Dette er hacken. Funksjonen tar egentlig inn `selectedDate` og `format`. Den er ment til å formatere enkeltdatoer. Dersom strengene som returneres her er like, vil Flatpickr ignorere den ene, og kun vise én.

Vi returnerer alltid "samme" streng to ganger per periode med denne funksjonen, men kun den ene vises. Slik bygger vi formatet helt selv :)